web.rb

```diff
@@ -36,9 +36,25 @@ def log_info(message)
   send_file 'index.html'
 end
 
+def validateApiKey
+  if Stripe.api_key.start_with?('pk')
```

Comment:
wdyt about also checking for `Stripe.api_key.nil?`

I think this code will crash as-is without it

Reply:
ah good call!